test/e2e/performanceprofile/testdata/render-expected-output/manual_tuned.yaml

```diff
@@ -54,7 +54,7 @@ spec:
       tuned.non_isolcpus=${not_isolated_cpumask} systemd.cpu_affinity=${not_isolated_cores_expanded}
       intel_iommu=on iommu=pt\n\n\ncmdline_isolation=+isolcpus=managed_irq,${isolated_cores}\n\n\n\ncmdline_realtime=+nohz_full=${isolated_cores}
       tsc=nowatchdog nosoftlockup nmi_watchdog=0 mce=off skew_tick=1 rcutree.kthread_prio=11\n\n\n\n\n\n\ncmdline_hugepages=+
-      default_hugepagesz=1G   hugepagesz=2M hugepages=128 \n\ncmdline_additionalArg=+\n"
+      default_hugepagesz=1G   hugepagesz=2M hugepages=128 \n\ncmdline_additionalArg=+\n\n\n\n\n"
```

Comment:
???

Reply:
When tuned template [assets/performanceprofile/tuned/openshift-node-performance](https://github.com/openshift/cluster-node-tuning-operator/pull/415/files/0c12451b26164b9edec463eac428b64494b0edd5#diff-e0d5dac27460da1c7759fdee98e915a6c1a4eed9a8969202562add4c6d41b292) the output string to check the test in tuned_test.go. If no \n are added, test fails.
I am thinking in a better way to test the changes in the template creating a struct parsing the yaml but it's not easy because the way template is parsed.

Reply:
I have this commit to improve tuned test https://github.com/marioferh/cluster-node-tuning-operator/commit/6eacd3c30f24323eda9f8e419fb5b57c05556f56

But the test above is about render command that the output is a string with the whole tuned kubernetes object in string format, I don't know even if this patch is ugly, if it worth the time to parse the whole string and check every field

Reply:
OK - but prior to your changes, the yaml file was being rendered without the extra newlines - right? So doesn't that mean there is something wrong with this PR in that it is now introducing a bunch of extra newlines?

Reply:
The issue is due because these lines:
`
{{if .HighPowerConsumption}}
cmdline_pstate=+intel_pstate=disable
{{end}}

{{if .PerPodPowerManagement}}
cmdline_pstate=+intel_pstate=passive
{{end}}
`

Before both are = false, the output of the test is: 

`                                          "data": strings.Join({                                                                                                                                                             
                                                ... // 4119 identical bytes                                                                                                                                                
                                                " rcutree.kthread_prio=11\n\n\n\n\n\n\ncmdline_hugepages=+ default_hugep",                                                                                                 
                                                "agesz=1G   hugepagesz=2M hugepages=128 \n\ncmdline_additionalArg=+",                                                                                                      
        -                                       "\n\n\n\n",                                                                                                                                                                
                                                "\n",                                                                                                                                                                      
                                        }, ""),                                                                                                                                                                            
                                        "name": string("openshift-node-performance-manual"),      `

I don't see another solution. 
And I've said before, it makes no sense for mi parser a output string of binary to an struct just for this test.